<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>838</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1277</width>
    <height>838</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="tbn_min">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tbn_max">
   <property name="geometry">
    <rect>
     <x>1175</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tbn_close">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn_close1">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn_max1">
   <property name="geometry">
    <rect>
     <x>1175</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn_min1">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wyy">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="min">
   <property name="geometry">
    <rect>
     <x>1085</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="min1">
   <property name="geometry">
    <rect>
     <x>1085</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tbn_mail">
   <property name="geometry">
    <rect>
     <x>1025</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn_mail1">
   <property name="geometry">
    <rect>
     <x>1025</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="separate">
   <property name="geometry">
    <rect>
     <x>1067</x>
     <y>20</y>
     <width>10</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn_setting1">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tbn_clothing">
   <property name="geometry">
    <rect>
     <x>935</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn_clothing1">
   <property name="geometry">
    <rect>
     <x>935</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="title_user">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>11</y>
     <width>206</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_user1">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>12</y>
     <width>206</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tbn_setting">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tbn_listening">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>18</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn_listening1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>18</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_searching1">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>18</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="title_searching">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>18</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="change_right1">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>22</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="change_right">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>22</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="change_left">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>22</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="change_left1">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>22</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cd">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>764</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pause1">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>762</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pause">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>762</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="last1">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="next1">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lyric">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lyric1">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="range">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="range1">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>760</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>790</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn2">
   <property name="geometry">
    <rect>
     <x>1181</x>
     <y>770</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tbn1">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>770</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="voice1">
   <property name="geometry">
    <rect>
     <x>1122</x>
     <y>774</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="voice">
   <property name="geometry">
    <rect>
     <x>1122</x>
     <y>774</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="voiceChange">
   <property name="geometry">
    <rect>
     <x>1137</x>
     <y>650</y>
     <width>10</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QSlider::groove:vertical
{
    width: 5px;
    background: #C0C0C0;
    border: 1px solid #4A708B;
    border-radius: 1px;
    padding-left:-1px;
    padding-right:-1px;
    padding-top:-1px;
    padding-bottom:-1px;
}

QSlider::sub-page:vertical
{
    background: rgb(232, 232, 232);
    border: 1px solid rgb(232, 232, 232);
    border-radius: 2px;
}

QSlider::add-page:vertical
{
    width: 10px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #c4c4c4, stop:1 #B1B1B1);
    background: qlineargradient(x1:0, y1:0.2, x2:1, y2:1, stop:0 #5DCCFF, stop: 1 #1874CD);
    border: 0px solid #777;
    border-radius: 2px;
}

QSlider::handle:vertical 
{
    height: 11px;
    background: qradialgradient
    (
        spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, 
        stop:0.6 #45ADED, stop:0.778409 rgba(255, 255, 255, 255)
    );
    margin-left: -3px;
    margin-right: -3px;
    border-radius: 5px;
}

QSlider::handle:vertical:hover
{  
    height: 11px;
    background: qradialgradient
    (
        spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, 
        stop:0.6 #2A8BDA, stop:0.778409 rgba(255, 255, 255, 255)
    ); 
    margin-left: -3px;
    margin-right: -3px;
    border-radius: 5px;
}

QSlider::sub-page:vertical:disabled
{
    background: #00009C;
    border-color: #999;
}

QSlider::add-page:vertical:disabled
{
    background: #eee;
    border-color: #999;
}

QSlider::handle:vertical:disabled
{
    background: #eee;
    border: 1px solid #aaa;
    border-radius: 4px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="voicechange">
   <property name="geometry">
    <rect>
     <x>1122</x>
     <y>630</y>
     <width>40</width>
     <height>137</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="starttime">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>810</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="endtime">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>810</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>817</y>
     <width>441</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
		border-radius: 5px;
        height: 10px;
}
QSlider::handle:horizontal {
        width: 3px;
		background:rgb(255, 0, 127);
        margin: -13px -1px -10px -1px;
}
QSlider::sub-page:horizontal{    
		background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(230, 230, 230), stop: 1 rgb(255, 0, 0));
}
QSlider::add-page:horizontal {
   		 background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(124, 124, 124), stop: 1.0 rgb(230, 230, 230));
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="last">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>773</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>voicechange</zorder>
  <zorder>tbn_min</zorder>
  <zorder>tbn_max</zorder>
  <zorder>tbn_close</zorder>
  <zorder>tbn_close1</zorder>
  <zorder>tbn_max1</zorder>
  <zorder>tbn_min1</zorder>
  <zorder>wyy</zorder>
  <zorder>min</zorder>
  <zorder>min1</zorder>
  <zorder>tbn_mail</zorder>
  <zorder>tbn_mail1</zorder>
  <zorder>separate</zorder>
  <zorder>tbn_setting1</zorder>
  <zorder>tbn_clothing</zorder>
  <zorder>tbn_clothing1</zorder>
  <zorder>title_user</zorder>
  <zorder>title_user1</zorder>
  <zorder>tbn_setting</zorder>
  <zorder>tbn_listening</zorder>
  <zorder>tbn_listening1</zorder>
  <zorder>title_searching1</zorder>
  <zorder>title_searching</zorder>
  <zorder>change_right1</zorder>
  <zorder>change_right</zorder>
  <zorder>change_left</zorder>
  <zorder>change_left1</zorder>
  <zorder>cd</zorder>
  <zorder>pause1</zorder>
  <zorder>pause</zorder>
  <zorder>last1</zorder>
  <zorder>next1</zorder>
  <zorder>lyric</zorder>
  <zorder>lyric1</zorder>
  <zorder>range</zorder>
  <zorder>range1</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>tbn2</zorder>
  <zorder>tbn1</zorder>
  <zorder>voice1</zorder>
  <zorder>voice</zorder>
  <zorder>voiceChange</zorder>
  <zorder>starttime</zorder>
  <zorder>endtime</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>next</zorder>
  <zorder>last</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
